manifests/init.pp

```diff
@@ -104,7 +104,18 @@
     }
   }
 
-  package { 'rubygem-foreman_scap_client': ensure => $ensure, install_options => $install_options, } ->
+  $scappackage = $operatingsystem ? {
+    ubuntu => rubygem-foreman-scap-client,
+    default => rubygem-foreman_scap_client
+  }
+
+  package { $scappackage: ensure => $ensure, install_options => $install_options, } ->
```

Comment:
Since you're renaming this, it'd be cleaner to also use the variable at line 103 in the `Package`.